### PR TITLE
Fixes #11460 - cleaner layout in  Host/Hostgroup parameters

### PR DESCRIPTION
**Pull request by amirfefer**

### CODE REVIEW

**Comment by dLobatog** [comment]
@amirfefer Thank you! This makes a huge difference on global parameters. I noticed a few glitches though if you want to align the whole thing.

In this picture - note how the values in Puppet classes overrides are not aligned. Just the value, titles are properly aligned. They are shifted to the right a few pixels.

screenshot from 2015-10-15 15-32-57

Aside from that, do you mind to update the commit description with a couple of screenshots showing the change, and a description of what changed? It's hard to know just by looking at the code.

**Comment by amirfefer** [comment]
Before:

parameters-1

After: 
parameters-2

The first change is in the puppet class overrides section, the fields are more aligned.
The second change is in the global parent parameters, the name field is  truncated  with tooltip , and changed from disabled textbox format.

**Comment by dLobatog** [comment]
Thanks for the update @amirfefer , we're almost there. Puppet class overrides are still a bit offset and unattached to the table headers. If you see all other tables, they are aligned.

screenshot from 2015-10-19 19-21-28

Aside from that, what I meant by 'updating the commit description' with screenshots and a description was updating the actual git commit description, not the pull request, sorry for being unclear. You can do that by just running `git commit --amend` and start writing your commit description 2 lines below the title.

**Comment by dLobatog** [comment]
Much better, thanks @amirfefer 

screenshot from 2015-10-30 13-44-30

**Comment by dLobatog** [comment]
Merged as 91851db017953aebb117c69802fb80a84800b925, thanks @amirfefer !

File: app/views/common_parameters/_parent_parameter.html.erb [hunk 1]
```diff
@@ -11,7 +11,7 @@
       <% next if @hostgroup.group_parameters.map(&:name).include? parameter.name %>
       <tr>
         <td>
-          <%= text_field_tag("parent_parameter_#{i}_key", '', :class => "form-control", :value => parameter.name, :readonly => true) %>
```
Comment (domcleal) [inline app/views/common_parameters/_parent_parameter.html.erb:14]: This text field is used in the controller when overriding these parameters, so now parent parameter overrides don't work.  #2649 refactors this and will fix it anyway.

**Pull request closed without merging**